### DISCUSSION TITLE
[Feature] Bulk Select Share in Album

### DISCUSSION THREAD

**Discussion by kflinderman** [immich-app/immich#3752] [Feature Request]
### The feature

It looks like you can bulk select share from the timeline in the app, but I was looking to do the same from an album.  I like to curate the mass amount of photos into albums and then share them via the Android OS Share tab.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This is now supported

**Reply by pablo03v** [reply]
> This is now supported

In the mobile app only? I can't seem to select multiple albums online yet :)

**Reply by bo0tzz** [reply]
This is selecting inside of the album viewer

**Reply by pablo03v** [reply]
> This is selecting inside of the album viewer

ohhh, inside of an album itself. I thought album viewer might be the view of the albums. sry
